### PR TITLE
chore: update template with other automatic links

### PR DESCRIPTION
**Pull request by hdd**
Resolves FTRACK-

- [ ] I have added automatic tests where applicable
- [ ] The PR title is suitable as a release note
- [ ] The PR contains a description of what has been changed
- [ ] The description contains manual test instructions

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+  Copy the id and paste it to the appropriate CLICKUP-<id> / FTRACK-<id> /  SENTRY-<id> / ZENDESK-<id> link.
+  Please remember to remove the unused ones.
 -->
+Resolves : 
 
-Resolves FTRACK-
+* CLICKUP-
+* FTRACK-
+* SENTRY-
+* ZENDESK-
```
Comment (jimmycallin) [inline .github/pull_request_template.md]: I think it might be better to keep all of these as part of the comment above, since I believe it's more annoying having to remove the ones that are very rarely used every time, than to add the tracking issues you want to use.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+- [ ] The PR contains update to the release notes.
+- [ ] The PR contains update to the documentation.
 
-## Changes
+<!--
+  What platform / browser this PR has been tested with.
+-->
 
+This PR has been tested on :
+- [ ]
```
Comment (jimmycallin) [inline .github/pull_request_template.md]: I don't think this is generic enough to be applicable as a default PR template description. Any repositories that wants to track which platsforms and browsers it has been tested on should add their own PR template imo.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
 
-- [ ] I have added automatic tests where applicable
-- [ ] The PR title is suitable as a release note
-- [ ] The PR contains a description of what has been changed
-- [ ] The description contains manual test instructions
+- [ ] I have added automatic tests where applicable.
+- [ ] The PR contains a description of what has been changed.
+- [ ] The description contains manual test instructions.
+- [ ] The PR contains update to the release notes.
+- [ ] The PR contains update to the documentation.
```
Comment (jimmycallin) [inline .github/pull_request_template.md]: In my mind, this list should be a todo list for the PR creator to make sure they have done all the necessary steps, and not a list that describes what has changed within the PR (since that is obvious from the file diffs). 

That is, when PRs are opened, this list should optimally be 100% checked every time. As such, I would rephrase the last point to:

```suggestion
- [ ] The PR contains updates to the release notes.
- [ ] I have verified that the documentation is still up to date.
```

**Review by jimmycallin — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (hdd) [reply .github/pull_request_template.md]: moved them into the comment, and added a line to express the need of adding one for reference

File: .github/pull_request_template.md (hunk 2 above)
Comment (hdd) [reply .github/pull_request_template.md]: thought the idea was to have templates to further customise based on specific needs (hence keeping this open wide) but sure.

File: .github/pull_request_template.md (hunk 3 above)
Comment (hdd) [reply .github/pull_request_template.md]: done now

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,26 +1,22 @@
...
   Copy the id and paste it to the appropriate CLICKUP-<id> / FTRACK-<id> /  SENTRY-<id> / ZENDESK-<id> link.
   Please remember to remove the unused ones.
+  
+  * CLICKUP-
+  * FTRACK-
+  * SENTRY-
+  * ZENDESK-
+
 -->
 Resolves :
```
Comment (jimmycallin) [inline .github/pull_request_template.md]: Remove space before :

**Review by jimmycallin — APPROVED** [review]

**Review by jimmycallin — APPROVED** [review]
